app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -456,11 +455,10 @@ def send_enrollment_status_sms_notification(enrollment:)
     sms_delay_hours = IdentityConfig.store.in_person_results_delay_in_hours ||
                       DEFAULT_EMAIL_DELAY_IN_HOURS
     wait_until = enrollment.proofed_at + sms_delay_hours
-    if wait_until > Time.zone.now
-      InPerson::SendProofingNotificationJob.set(wait_until: wait_until).perform_later(enrollment.id)
-    else
-      InPerson::SendProofingNotificationJob.perform_later(enrollment.id)
-    end
+    InPerson::SendProofingNotificationJob.set(
+      wait_until: wait_until,
+      queue: :intentionally_delayed,
```

Comment:
we can also set the `queue_as` at the class-level for this job

https://github.com/18F/identity-idp/blob/0f21a2a06f467873b6391dd20621d50c90b4e633/app/jobs/resolution_proofing_job.rb#L4